python/instrumentation/openinference-instrumentation-beeai/pyproject.toml

```diff
@@ -33,10 +33,10 @@ dependencies = [
 
 [project.optional-dependencies]
 instruments = [
-  "beeai-framework[duckduckgo,wikipedia]==0.1.18",
+  "beeai-framework >= 0.1.10",
 ]
 test = [
-  "beeai-framework[duckduckgo,wikipedia]==0.1.18",
```

Comment:
I removed the qualification here because we don't actually instrument the duckduckgo or wikipedia code at the moment.